packages/babel-plugin-proposal-object-rest-spread/src/index.js

```diff
@@ -17,14 +17,15 @@ const ZERO_REFS = (() => {
 export default declare((api, opts) => {
   api.assertVersion(7);
 
-  const { useBuiltIns = false, loose = false } = opts;
-
-  if (typeof loose !== "boolean") {
```

Comment:
Why removing loose type check here?

Reply:
I wrongly assumed that `loose` wasn't used anymore :upside_down_face: